lametro/templatetags/lametro_extras.py

```diff
@@ -2,6 +2,7 @@
 from django.template.defaultfilters import stringfilter
 from django.utils.html import strip_entities, strip_tags
 from django.utils import timezone
+from django.core.exceptions import MultipleObjectsReturned
```

Comment:
this totally works, just a fun trivia bit: you can access this and the `ObjectDoesNotExist` exceptions directly from any django model, like `LAMetroEvent.MultipleObjectsReturned`.